packages/video_player/tizen/src/video_player.cc

```diff
@@ -8,6 +8,7 @@
 #include <flutter/standard_method_codec.h>
 
 #include <algorithm>
+#include <queue>
```

Comment:
`#include <queue>` can be removed from `video_player.h`,

Reply:
I have removed it from header file.